lib/spree/chimpy/controller_filters.rb

```diff
@@ -3,8 +3,8 @@ module ControllerFilters
     extend ActiveSupport::Concern
 
     included do
-      before_filter :set_mailchimp_params
-      before_filter :find_mail_chimp_params, if: :mailchimp_params?
+      before_action :set_mailchimp_params
```

Comment:
@thogg4 thanks for these bumps! You still looking to get this merged into master?

Reply:
@braidn yep

Reply:
Do you want this just in master or another branch?

Reply:
@braidn master

Reply:
thanks @braidn